LWA_EPIC/LWA_EPIC.py

```diff
@@ -754,7 +754,7 @@ def main(self):
                 if nchan % self.nchan_out == 0:
```

Comment:
A small suggestion--I think it'd be good if we add an additional check for ```nchan_out>6``` here to avoid integer overflow during averaging. It's highly unlikely that we'll use less than 7 channels to cover the entire subband, but just in case.

Reply:
What happens when `nchan_out<7`?

Reply:
If we set ```nchan_out``` to 4, for example, we average 33 channels and their sum can be greater than 255. This leads to an overflow in the ```re``` or ```im``` variables in the averaging function that is launched on the gpu.